src/components/Header.astro

```diff
@@ -16,16 +16,15 @@ import {GITHUB, MY_NAME, SITE_TITLE} from "../config";
       <ul class="unset flex gap-4 [&>li]:p-0">
         <!--<li><HeaderLink href="/">Blog</HeaderLink></li>-->
         <li><HeaderLink href="/">Resume</HeaderLink></li>
-        <li><HeaderLink href="/experience">Additional Info</HeaderLink></li>
         <li><HeaderLink href="/projects">Projects</HeaderLink></li>
-
+        <li><HeaderLink href="/experience">Additional Info</HeaderLink></li>
```

Comment:
Move projects closer to resume so that they get to see the cool engine / Vulkan work you've done quicker